tests/test_django.py

```diff
@@ -40,7 +40,7 @@ def test_fields(mixer):
     assert isinstance(rabbit.object_id, int)
     assert rabbit.object_id >= 0
     assert isinstance(rabbit.error_code, int)
-    assert rabbit.error_code >= 0
```

Comment:
Instead of removing this line all together, leave both bare assertions.  
The `PositiveSmallIntegerField` needs both positivity and a value under `32768`.
```
assert rabbit.error_code >= 0
assert rabbit.error_code <= 32767
```